### PR TITLE
Completed javascript-constructors Exercise - Bingle Padilla

### PR DESCRIPTION
**Pull request by codewithbingle**
javascript-constructor
javascript-constructor-example-contructor
javascript-constructor-student js

### CODE REVIEW

File: javascript-constructors/example-constructor.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function ExampleConstructor() {}
+
+console.log('value of ExampleConstructor.prototype: ', ExampleConstructor.prototype);
+console.log('type of ExampleConstructor.prototype: ', ExampleConstructor.prototype);
```
Comment (TimelikeClosure) [inline javascript-constructors/example-constructor.js]: The second one should be its type, not its value

File: javascript-constructors/new-objects.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+console.log('value of aString: ', aString);
+console.log('typeof aString: ', aString);
+
+var aNumber = new Number();
+console.log('value of aNumber: ', aNumber);
+console.log('typeof aNumber: ', aNumber);
+
+var aBoolean = new Boolean();
+console.log('value of aBoolean: ', aBoolean);
+console.log('typeof aBoolean: ', aBoolean);
```
Comment (TimelikeClosure) [inline javascript-constructors/new-objects.js]: Please output the type of each value.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: javascript-constructors/example-constructor.js (hunk 1 above)
Comment (codewithbingle) [reply javascript-constructors/example-constructor.js]: Noted.

Comment (codewithbingle) [reply javascript-constructors/example-constructor.js]: javascript-contructors-example-constructors-2

File: javascript-constructors/new-objects.js (hunk 2 above)
Comment (codewithbingle) [reply javascript-constructors/new-objects.js]: javascript-contructors-new-objects-2

**Comment by codewithbingle** [comment]
javascript-contructors-example-constructors-2
javascript-contructors-new-objects-2

**Comment by codewithbingle** [comment]
Updated.

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**